apps/docs/pages/guides/platform/terraform/[[...slug]].tsx

```diff
@@ -0,0 +1,149 @@
+import { CodeHikeConfig, remarkCodeHike } from '@code-hike/mdx'
+import matter from 'gray-matter'
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next'
+import { MDXRemote } from 'next-mdx-remote'
+import { serialize } from 'next-mdx-remote/serialize'
+import remarkGfm from 'remark-gfm'
+import rehypeSlug from 'rehype-slug'
+
+import codeHikeTheme from 'config/code-hike.theme.json' assert { type: 'json' }
+
+import components from '~/components'
+import Layout from '~/layouts/DefaultGuideLayout'
+import { isValidGuideFrontmatter } from '~/lib/docs'
+import { UrlTransformFunction, linkTransform } from '~/lib/mdx/plugins/rehypeLinkTransform'
+import remarkMkDocsAdmonition from '~/lib/mdx/plugins/remarkAdmonition'
+import { removeTitle } from '~/lib/mdx/plugins/remarkRemoveTitle'
+import remarkPyMdownTabs from '~/lib/mdx/plugins/remarkTabs'
+
+// We fetch these docs at build time from an external repo
+const org = 'supabase'
+const repo = 'terraform-provider-supabase'
+const branch = 'main'
+const docsDir = 'docs'
+
+// Each external docs page is mapped to a local page
+const pageMap = [
+  {
+    remoteFile: 'index.md',
```

Comment:
Are we planning to add the resource specific docs too? https://github.com/supabase/terraform-provider-supabase/tree/main/docs/resources

Reply:
Oops I completely missed you commenting on this PR! Must've gotten buried in my notifs, sorry! 🤦🏼‍♀️ 

These are the ones I wanted the JSON schema for! So we have more flexibility in the display. Now that we have it I'll play around and see how to best assemble them 😄